src/plugins/projectGalaxy/components/CustomBlock.vue

```diff
@@ -129,7 +129,8 @@ export default {
         this.oats = this.space.plugins.projectGalaxy.oats;
         this.currentCampaignUrl = this.oats[this.proposal.id];
         if (this.currentCampaignUrl) {
-          this.currentCampaignId = this.currentCampaignUrl.match(/[^/]\w+/g)[2];
+          this.currentCampaignId =
+            this.currentCampaignUrl.match(/[^/]+(?=\/$|$)/g)[0];
```

Comment:
I love regex but I also love readability. A simple `split('/').pop()` would've done the trick here I guess. 😃

Reply:
yeah, `split('/').pop()` will be more elegant

Reply:
but maybe next time we will do the optimization?